### PR TITLE
[Hub Convert] Cef, Chaosreader, Chardet, checkdmarc, checkmarx CxSAST, checkpoint sand blast, Cherwell, Cif

### PR DESCRIPTION
**Pull request by lmilby-r7**
## Proposed Changes

Describe the proposed changes:

  -

## PR Requirements

Developers, verify you have completed the following items by checking them off:

### Style

Review the [style guide](https://komand.github.io/python/style.html)

- [ ] For dependencies, pin [OS package](https://komand.github.io/python/style.html#dockerfile) and [Python package](https://komand.github.io/python/style.html#requirements-txt) versions
- [ ] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [ ] For size, use the [slim SDK images](https://komand.github.io/python/sdk.html#sdk-versions) when possible: ``komand/python-3-37-slim-plugin`` and ``komand/python-3-37-plugin``
- [ ] For error handling, use of [PluginException](https://komand.github.io/python/error_handling.html#plugin-exceptions) and [ConnectionTestException](https://komand.github.io/python/error_handling.html#connection-exceptions)
- [ ] For logging, use [self.logger](https://komand.github.io/python/sdk.html#logging)
- [ ] For docs, use [changelog style](https://komand.github.io/python/style.html#changelog)
- [ ] For docs, validate markdown with ``make validate`` which calls ``mdl`` to lint ``help.md``

### Functional Checklist
- [ ] Work fully completed
- [ ] Functional
  - [ ] Any new actions/triggers include JSON [test files](https://komand.github.io/python/style.html#tests) in the `tests/` directory created with `./run -c sample $action > tests/$action.json`
  - [ ] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [ ] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [ ] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `./run -T tests/example.json -d -j`
    * All actions/triggers shortcut `./run -T all -d -j` (use PR format at end)
  - [ ] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `./run -R tests/example.json -d -j`
    * All actions/triggers shortcut `./run -R all -d -j` (use PR format at end)

### Assessment

You must validate your work to reviewers:

1. Run `make validate` and make sure everything passes
2. Run the assessment tool: `./run -A -R all -T all`. For single action validation: `./run -A -R tests/my_action.json -T tests/my_action.json`
3. Copy (`./run ... | pbcopy`) and paste the output in **a new post** on this PR.
4. Add UI screenshot of the workflow used for testing
5. Add UI screenshot of the job output used for testing
6. Add UI screenshot of the artifact (See rules in UI Checklist) used for testing

### CODE REVIEW

File: cef/help.md [hunk 1]
```diff
@@ -1,25 +1,21 @@
 # Description
 
-Common Event Format (CEF) is an extensible, text-based format that defines a syntax for log records comprised of a
```
Comment (jschipp-r7) [inline cef/help.md]: There is some good information in the original that we should build off of. If anything keep that, and add your sentence after.

File: cef/plugin.spec.yaml [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
 extension: plugin
-product: ["insightconnect"]
+product: [insightconnect]
 name: cef
 title: CEF
 vendor: rapid7
 support: community
 status: []
-description: Work with Common Event Format (CEF) data
+description: This plugin used for generating and manipulating event in a Common Event Format (CEF).
```
Comment (jschipp-r7) [inline cef/plugin.spec.yaml]: No period, DescriptionValidator will lint on that

File: cef/help.md (hunk 1 above)
Comment (jschipp-r7) [reply cef/help.md]: Same for the others

File: cef/plugin.spec.yaml (hunk 2 above)
Comment (jschipp-r7) [reply cef/plugin.spec.yaml]: Same for any others

File: chardet/help.md [hunk 3]
```diff
@@ -1,17 +1,14 @@
 # Description
 
-[Chardet](https://chardet.readthedocs.io/en/latest/index.html) is a python compatible character encoding detector.
+This plugin is a Python compatible character encoding detector.
```
Comment (mrinehart-r7) [inline chardet/help.md]:
```suggestion
[Chardet](https://chardet.readthedocs.io/en/latest/index.html) is a Python-compatible character encoding detector. Using this plugin within an automation workflow can assist with service interoperability and other text manipulation needs.
```

File: chardet/plugin.spec.yaml [hunk 4]
```diff
@@ -3,20 +3,21 @@ extension: plugin
 product: ["insightconnect"]
 name: chardet
 title: Chardet
-description: Python compatible character encoding detector
+description: This plugin is a Python compatible character encoding detector
```
Comment (mrinehart-r7) [inline chardet/plugin.spec.yaml]:
```suggestion
description: Chardet is a Python-compatible character encoding detector, useful for text manipulation or processing.
```

File: chardet/plugin.spec.yaml [hunk 5]
```diff
@@ -3,20 +3,21 @@ extension: plugin
...
 version: 1.0.0
 vendor: rapid7
 support: community
 status: []
 resources:
   source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/chardet
   license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url:
```
Comment (mrinehart-r7) [inline chardet/plugin.spec.yaml]:
```suggestion
  vendor_url: https://chardet.github.io/
```

**Pull request merged**